Windows Python Use AnatomicalOrientation::FromEnum type warning

Since the AnatomicalOrientation merge, the Azure Python wrapping build for `master` is failing do to the warning:

> 2024-11-12T23:18:45.4244116Z D:\a\1\s-build\Wrapping\Modules\ITKCommon\itkAnatomicalOrientationPython.cpp(3971): warning C4996: 'itk::AnatomicalOrientation::AnatomicalOrientation': Use the AnatomicalOrientation::FromEnum type instead.

@blowekamp should we just not emit the warning when building the Python wrapping?

--- a/Modules/Core/Common/include/itkAnatomicalOrientation.h
+++ b/Modules/Core/Common/include/itkAnatomicalOrientation.h
@@ -438,7 +438,7 @@ public:
    *
    * @param legacyOrientation
    */
-#  if defined(ITK_LEGACY_REMOVE) && !defined(ITK_LEGACY_SILENT)
+#  if defined(ITK_LEGACY_REMOVE) && !defined(ITK_LEGACY_SILENT) && !defined(ITK_WRAPPING)
   [[deprecated("Use the AnatomicalOrientation::FromEnum type instead.")]]
 #  endif
   AnatomicalOrientation(LegacyOrientationType legacyOrientation);

